scripts/helpers/utilities.mk

```diff
@@ -377,7 +390,10 @@ define prepare_loader_cache
         echo "/usr/local/lib"                 >> "$(DYNAMIC_LOAD)/sys32.ld.conf"; \
     fi; \
     if [[ -n "$(TARGET64)" ]]; then \
-        echo "$(AT_DEST)/$(TARGET)/lib64" >  "$(DYNAMIC_LOAD)/at64.ld.conf"; \
+        if [[ $1 -eq 1 ]]; then \
+            echo "$(AT_DEST)/slib-override"   >  "$(DYNAMIC_LOAD)/at64.ld.conf"; \
+        fi; \
+        echo "$(AT_DEST)/$(TARGET)/lib64" >> "$(DYNAMIC_LOAD)/at64.ld.conf"; \
```

Comment:
Is there another mechanism removing `$(DYNAMIC_LOAD)/at64.ld.conf` between executions 1 and 2?

I'm asking because I'm afraid this code won't generate correct results if that doesn't happen, i.e. the line with slib-override will still be there and `$(AT_DEST)/$(TARGET)/lib64` will be duplicated.

Reply:
I took me a minute, but I understand you have noticed that the `>` operation is within the `if` clause, so the file will only be appended during execution 2. Let me make sure that the file is cleared / recreated during the second run.